app/views/devise/registrations/new.html.haml

```diff
@@ -6,6 +6,7 @@
         = image_tag("default.png", :alt => "profile", :class => 'profile-img')
         = form_for(resource, as: resource_name, url: registration_path(resource_name),:html => { :class => "form-signin" }) do |f|
           = devise_error_messages!
+          = hidden_field_tag "user[roles][]", "viewer"
```

Comment:
should this really be set here? potentially someone could just use postman and register as admin?